third_party/2and3/pynamodb/models.pyi

```diff
@@ -29,9 +29,9 @@ class Model(metaclass=MetaModel):
     @classmethod
     def batch_write(cls: Type[_T], auto_commit: bool = ...) -> BatchWrite[_T]: ...
     def delete(self, conditional_operator: Optional[Text] = ..., **expected_values) -> Any: ...
-    def update(self, attributes: Dict[Text, Dict[Text, Any]], conditional_operator: Optional[Text] = ..., **expected_values) -> Any: ...
+    def update(self, attributes: Optional[Dict[Text, Dict[Text, Any]]], actions: Optional[List[Any]], condition: Optional[Any], conditional_operator: Optional[Text] = ..., **expected_values) -> Any: ...
     def update_item(self, attribute: Text, value: Optional[Any] = ..., action: Optional[Text] = ..., conditional_operator: Optional[Text] = ..., **expected_values): ...
-    def save(self, conditional_operator: Optional[Text] = ..., **expected_values) -> Dict[str, Any]: ...
+    def save(self, condition=Optional[Any], conditional_operator: Optional[Text] = ..., **expected_values) -> Dict[str, Any]: ...
```

Comment:
do you mean `condition: Optional[Any] = ...`?

Reply:
Yes, updated the code to fix it.